ocaml/xapi/xapi_globs.ml

```diff
@@ -1183,6 +1183,9 @@ module Resources = struct
     ; ( "generate_ssl_cert"
       , Constants.generate_ssl_cert
       , "script to generate SSL certificates to be used by XAPI" )
```

Comment:
This needs updating?

Reply:
I think we should update the doc string (and perhaps rename `generate_ssl_cert` -> `gencert`)